### PR TITLE
Create 2021-02-15-contributing-to-solidity-101.md

### PR DESCRIPTION
**Pull request by franzihei**
- TBD: Inlude running tests / fuzzing --> yes/no?

### CODE REVIEW

File: _posts/2021-02-15-contributing-to-solidity-101.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+In this post, we will walk you through the various different ways how you can get involved contributing to Solidity! Do not hesitate to contact us in case anything is left unclear after reading the post.
+
+### Overview of Contribution Opportunities  
+
+There are plenty of options how you can contribute to Solidity.  
+
+In particular, we appreciate support in the following areas:
+
+- **Issues**: Help with reporting or fixing issues.
+- **Testing**: Run the compiler tests or help with fuzz testing.
```
Comment (chriseth) [inline _posts/2021-02-15-contributing-to-solidity-101.md:21]: Ah, now I understand! I guess running the compiler tests is not that helpful, but probably something like "testing out new compiler versions, preferrably nightly versions, on your code".

Comment (franzihei) [reply _posts/2021-02-15-contributing-to-solidity-101.md:21]: Ok perfect, will add that (and remove the "run the tests bit")! What about helping with fuzzing? https://docs.soliditylang.org/en/v0.8.1/contributing.html#running-the-fuzzer-via-afl

**Review by chriseth — COMMENTED** [review]
Apart from suggesting to try out nightly versions of the compiler, this looks very good!

**Comment by franzihei** [comment]
Can I publish this today @chriseth?

**Pull request merged**